source/content.css

```diff
@@ -920,3 +920,21 @@ body > .footer li a {
 .rgh-files-hidden .file-wrap {
 	border: 0;
 }
+
+
+.notifications-list .boxed-group-action {
+	margin: 5px 6px;
+}
+
+.notifications-list .boxed-group-action button {
+	padding: 3px 10px;
+	border: 0;
+	margin-left: 2px;
+	color: #586069;
+	background: none;
+}
+
+.notifications-list .boxed-group .text-green {
+	display: inline-block;
```

Comment:
What is this for? It's breaking another GitHub extension of mine so if necessary we might need to make it more specific (i.e. prepend `#notification-center`)

Reply:
Related note: perhaps this can just be replaced by code that hides the _Open tabs_ button when the `text-green` is shown, because once that's shown the button is useless:

![a](https://user-images.githubusercontent.com/1402241/35935335-c1e2a160-0c72-11e8-9e85-3a796b97b5dd.gif)